docs/tools/flow-cli/flix.md

```diff
@@ -38,21 +77,20 @@ For a list of all templates, check out the [FLIX template repository](https://gi
 
 To generate a FLIX, see the [FLIX CLI readme](https://github.com/onflow/flow-interaction-template-tools/tree/master/cli).
 
-## Arguments
+## Arguments for both execute and package
```

Comment:
not following "title case", Flags for both execute and package is the title

Reply:
Title case is a style of capitalization which would match the style of the rest of the docs. If it was title case it would be "Arguments for Both Execute and Package"